visualizer/filters-bar.css

```diff
@@ -0,0 +1,31 @@
+#filters-bar{
```

Comment:
A little nit, but would be good to keep the spacing between selector and curly brace across all this new CSS 👍

Reply:
😜